Dronuts/common/models/order.json

```diff
@@ -1,16 +1,12 @@
 {
   "name": "Order",
   "plural": "Orders",
-  "base": "Model",
+  "base": "PersistedModel",
   "idInjection": true,
   "options": {
     "validateUpsert": true
   },
   "properties": {
-    "order_id": {
```

Comment:
Why is `order_id` being removed? Don't we need an ID to track the items?

Reply:
Since we added a mongo database and changed the models from Model to PersistedModel, the ID field is added automatically. We no longer need order_id as a manually created property.